src/spnavd.h

```diff
@@ -23,7 +23,11 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "cfgfile.h"
 #include "logger.h"
 
+#if defined(__FreeBSD__)
+#define DEF_CFGFILE		"/usr/local/etc/spnavrc"
+#else
 #define DEF_CFGFILE		"/etc/spnavrc"
+#endif
 #define DEF_LOGFILE		"/var/log/spnavd.log"
```

Comment:
I'm not sure about this. `/usr/local` should not be hardcoded. The best way to handle this is to add a `configure` option for the config file path. I'll handle this in the master branch, drop this change, leave it to `/etc` for now.